src/quo2/components/list_items/account_list_card/style.cljs

```diff
@@ -0,0 +1,20 @@
+(ns quo2.components.list-items.account-list-card.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container
+  [state theme]
+  (let [light-bg     (if (= state :default) colors/neutral-2-5 colors/neutral-5)
+        dark-bg      (if (= state :default) colors/neutral-80-opa-40 colors/neutral-80-opa-60)
+        light-border (if (= state :default) colors/neutral-10 colors/neutral-20)
+        dark-border  (if (= state :default) colors/neutral-80 colors/neutral-70)]
```

Comment:
Just as a personal preference i would use a single `if`, this doesn't affect anything though.

Reply:
@rasom Like this ?

```
(defn container
  [state theme]
  (let [[light-bg dark-bg light-border dark-border]
        (if (= state :default)
          [colors/neutral-2-5 colors/neutral-80-opa-40 colors/neutral-10 colors/neutral-80]
          [colors/neutral-5 colors/neutral-80-opa-60 colors/neutral-20 colors/neutral-70])]
...
```

IMO this is less readable